production/ksonnet/loki/multi-zone.libsonnet

```diff
@@ -21,7 +21,7 @@
     multi_zone_ingester_migration_enabled: false,
     multi_zone_ingester_replication_write_path_enabled: true,
     multi_zone_ingester_replication_read_path_enabled: true,
-    multi_zone_ingester_replicas: 0,
+    multi_zone_ingester_replicas: 3,
```

Comment:
wdyt of only having 3 minimum replicas if it is enabled?

Reply:
Because without this, there is no Ingester Pod deployed.
3 replicas would allow having 1 Pod per zone.

Reply:
@DylanGuedes do you need anything else to merge this PR?

Reply:
@DylanGuedes sounds good thanks.

I was also thinking that there is no doc available about what multi zone Ingesters are, how to manage them and how to migrate.

Reply:
Could you create an issue about that? We should definitely cover it.

Reply:
I'm going to do that, thanks

Reply:
See  #9199